test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Telemetry/RestoreTelemetryServiceTests.cs

```diff
@@ -54,6 +54,15 @@ public void RestoreTelemetryService_EmitRestoreEvent_OperationSucceed(bool force
                 packageCount: 2,
                 noOpProjectsCount: noopProjectsCount,
                 upToDateProjectsCount: 5,
+                unknownProjectsCount: 0,
```

Comment:
You should make the correct values are assigned to the correct item.

Reply:
Ok, I need to provide more test coverage here.